ui/raidboss/data/07-dt/raid/r7s.txt

```diff
@@ -1,9 +1,323 @@
+39.1 "--sync--" Ability { id: "A55A", source: "Brute Abombinator" }
+43.1 "Spore Sac (cast)" Ability { id: "A569", source: "Brute Abombinator" }
+46.2 "--sync--" Ability { id: "A516", source: "Brute Abombinator" }
+48.2 "Spore Sac" Ability { id: "A56A", source: "Brute Abombinator" }
+52.6 "Sinister Seeds (cast)" Ability { id: "A56D", source: "Brute Abombinator" }
+53.7 "Pollen" Ability { id: "A56B", source: "Brute Abombinator" }
+55.8 "--adds targetable--"
+56.6 "Sinister Seeds (puddles 1)" #Ability { id: "A571", source: "Brute Abombinator" }
+58.6 "Sinister Seeds (puddles 2)" #Ability { id: "A571", source: "Brute Abombinator" }
+60.6 "Sinister Seeds (seeds drop)" Ability { id: "A56E", source: "Brute Abombinator" }
+60.6 "Sinister Seeds (puddles 3)" #Ability { id: "A571", source: "Brute Abombinator" }
+62.6 "Sinister Seeds (puddles 4)" #Ability { id: "A571", source: "Brute Abombinator" }
+65.2 "Impact" Ability { id: "A573", source: "Brute Abombinator" }
+65.2 "Tendrils of Terror" Ability { id: ["A56F", "A570"], source: "Brute Abombinator" }
+70.5 "Roots of Evil" Ability { id: "A572", source: "Brute Abombinator" }
+75.2 "--sync--" Ability { id: "A55A", source: "Brute Abombinator" }
+78.2 "--sync--" Ability { id: "A55A", source: "Brute Abombinator" }
+78.2 "Winding Wildwinds (adds)" #Ability { id: "A90D", source: "Blooming Abomination" }
+78.2 "Crossing Crosswinds (adds)" #Ability { id: "A90E", source: "Blooming Abomination" }
+81.2 "--sync--" Ability { id: "A55A", source: "Brute Abombinator" }
+84.2 "--sync--" Ability { id: "A55A", source: "Brute Abombinator" }
+87.2 "--sync--" Ability { id: "A55A", source: "Brute Abombinator" }
+90.0 "Hurricane Force (adds enrage)" #Ability { id: "A56C", source: "Blooming Abomination" }
+93.2 "Quarry Swamp" Ability { id: "A575", source: "Brute Abombinator" }
+94.2 "--sync--" Ability { id: "A55A", source: "Brute Abombinator" }
+97.2 "--sync--" Ability { id: "A55A", source: "Brute Abombinator" }
+100.2 "--sync--" Ability { id: "A55A", source: "Brute Abombinator" }
+103.2 "--sync--" Ability { id: "A55A", source: "Brute Abombinator" }
+105.2 "Explosion 1" Ability { id: "A576", source: "Brute Abombinator" }
+107.8 "Explosion 2" #Ability { id: "A576", source: "Brute Abombinator" }
+108.4 "Stoneringer" Ability { id: ["A55D", "A55E"], source: "Brute Abombinator" }
+110.4 "Explosion 3" #Ability { id: "A576", source: "Brute Abombinator" }
+117.3 "Smash Here/Smash There" Ability { id: ["A55F", "A560"], source: "Brute Abombinator" }
+118.3 "Brutish Swing" Ability { id: ["A561", "A562"], source: "Brute Abombinator" }
+119.4 "Brutal Smash" Ability { id: ["A565", "A566"], source: "Brute Abombinator" }
+121.4 "--sync--" Ability { id: "A55A", source: "Brute Abombinator" }
+127.1 "Pulp Smash" Ability { id: "A579", source: "Brute Abombinator" }
+129.1 "It Came from the Dirt" Ability { id: "A57A", source: "Brute Abombinator" }
+129.1 "The Unpotted" #Ability { id: "A57B", source: "Brute Abombinator" }
```

Comment:
Here and elsewhere, we can probably elide `It Came From The Dirt`, since both abilities go off at the same time, and I would guess that most users don't know the names of the Protean components.